srcpkgs/freerdp3/template

```diff
@@ -34,18 +35,18 @@ makedepends="alsa-lib-devel cups-devel fuse3-devel icu-devel libXcursor-devel
  $(vopt_if ffmpeg ffmpeg6-devel cairo-devel) $(vopt_if sdl2 SDL2_ttf-devel)
  $(vopt_if smartcard 'pcsclite-devel pkcs11-helper-devel')
  $(vopt_if wayland 'libxkbcommon-devel wayland-devel')
- $(vopt_if webview libwebkit2gtk41-devel)"
+ $(vopt_if webview libwebkit2gtk41-devel) $(vopt_if pam pam-devel)"
 short_desc="Free RDP (Remote Desktop Protocol) client"
 maintainer="chrysos349 <chrysostom349@gmail.com>"
 license="Apache-2.0"
 homepage="https://www.freerdp.com/"
 changelog="https://raw.githubusercontent.com/FreeRDP/FreeRDP/master/ChangeLog"
 distfiles="https://github.com/FreeRDP/FreeRDP/archive/${version}.tar.gz"
-checksum=ad31fdacf41751ef5a625c8f8cfef239411fc62805236fa1d422cf4bf5d0e190
+checksum=04a2e8049602e7aba767880990959facf2ebf7b5fbeeb5120e81838d8c2d2f17
 
 # wayland - deprecated, sdl2 - experimental
-build_options="aad ffmpeg sdl2 smartcard wayland webview"
-build_options_default="aad ffmpeg sdl2 smartcard"
+build_options="aad ffmpeg pam sdl2 smartcard wayland webview"
+build_options_default="aad ffmpeg pam sdl2 smartcard"
```

Comment:
Just hard depend on PAM. Build options are lousy UX in general and an option to build without PAM won't really save anything.

Reply:
Also, why disable wayland? and webview?